### PR TITLE
Remove duplicated authors

### PR DESCRIPTION
**Pull request by skotniczny**
It fixes a bug of duplicating authors, which occurs if there are more than two authors of the book.

For example:

http://lubimyczytac.pl/ksiazka/185218/html5-i-css3-zaawansowane-wzorce-projektowe
```
{ 
       title: 'HTML5 i CSS3. Zaawansowane wzorce projektowe',
       author: 'Michael Bowers, Dionysios Synodinos, Michael Bowers, Dionysios Synodinos, Victor Sumner',
       isbn: '9788324644711',
       publishingDate: undefined
}
```

### CODE REVIEW

**Comment by msciborski** [comment]
Thanks for revewing code. Do you think that, good solutions could be creating another function called for example 'filterDuplicates'? Code will be more readable.

File: LubimyCzytacFetch.js [hunk 1]
```diff
@@ -74,7 +74,11 @@ const fetchBook = async (bookUrl) => {
     title = $('h1[itemprop="name"]').text();
   }
 
-  const author = $('.author-info-container a[itemprop="name"]').map((_, el) => $(el).text()).get().join(' ,');
+  let authorsList = $('.author-info-container a[itemprop="name"]').map((_, el) => $(el).text()).get();
+  if (authorsList.length > 2) {
```
Comment (msciborski) [inline LubimyCzytacFetch.js:78]: I'm taking about this part of commit.

**Review by msciborski — COMMENTED** [review]
I gonna do it. Thanks again!

**Comment by msciborski** [comment]
Pull-request accepted.

**Comment by skotniczny** [comment]
Yeah, it would be nice to add some utility functions, especially if the further development of the application will need it.

**Pull request merged**